### ISSUE TITLE
Add support for mobile devices

### ISSUE THREAD

**Issue by Fleker** [mark-when/obsidian-plugin#18]
### What would you like to be added?

I enjoy using Markwhen on my PC. I went to download the Obsidian Plugin on my phone and it told me that it was not supported on that device.

The Markwhen website works on my phone, maybe not perfectly, but well enough that it should be minimal effort to allow phones to support this plugin as well.

### Why is this needed?

I like how Obsidian is offline-first, as it lets me make changes on the fly while in places without cell service. As much as the web app works, having something that works with my phone directly would be a great tool.

**Comment by rounakdatta** [comment]
Indeed, this is a lovely plugin, and I'm using this to chart out my travel plans for the year. It'd be awesome to have mobile support. Developers - please let us know how we can contribute and make this happen.

PS: Just in case someone in a similar boat stumbles here - I tried multiple plugins for this purpose. While the Mermaid plugin is great, the zoom/expand capabilities are absent. I also tried spreadsheets plugin as well as excalidraw plugin, and no other plugin fit better than this MarkWhen.

**Comment by keoshi** [comment]
Similar thoughts: loving this plugin and yet I can’t edit my timelines on the go.

This would be neatly solved if .md files were used instead, it seems. Related issue: https://github.com/mark-when/obsidian-plugin/issues/6

**Comment by joshp12345** [comment]
Chiming in to request this as well :)

**Comment by alexguo78** [comment]
Many thanks to the author for changing .mw to .md.

**Comment by guillermomf2001** [comment]
I have decided to use obsidian with markwhen to organize my projects, activities... and the rest of my life :-D

But to get it I need that markwhen obsidian plugin works with Android.

So, is planned to give support to Android for this plugin in the future? Preferably soon 🤓

**Comment by Jose-Verdu-Diaz** [comment]
I would also love to see this plugin implemented for mobile devices! Are the developers planning to work on this feature? If not, would the developers be happy with receiving some external help in implementing this feature?

**Comment by kochrt** [comment]
I would love help. I don't even know how difficult it would be, could be super simple 🤷

**Comment by Fannyc96** [comment]
same request for mobile support too! it would be super useful
